general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+=======================
+Import CSV file to Odoo
+=======================
+
+
+
+Frequently Asked Questions
+--------------------------
+
+
+Need to import data from an other application?
+----------------------------------------------
+
+In order to re-create relationships between different records, you should use the unique identifier from the original application and map it to the **ID** (External ID) column in Odoo. When you import an other record that links to the first one, use **XXX/ID** (XXX/External ID) to the original unique identifier.
+
+The **ID** (External ID) will also be used to update the original import if you need to re-import modified data later, it's thus good practice to specify it whenever possible.
+
+
+
+I cannot find the field I want to map my column to?
```

Comment:
Adding a question mark does not make it a question. This sounds so frenchy I can smell the cheese ;-)